lib/msal-browser/src/cache/InMemoryStorage.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IWindowStorage } from "./IWindowStorage";
+
+export class InMemoryStorage implements IWindowStorage {
+
+    private cache: Record<string, string>;
+
+    constructor() {
+        this.cache = {};
```

Comment:
Could this be a map?

Reply:
Please make this a map. There is a PR I plan to work on to make all `Record<key, value>` references to `Map` references in our code, in increments. So any new data we are adding which resembles a map, let's make it a map.

Reply:
done.